app/views/layouts/_header.html.erb

```diff
@@ -3,7 +3,7 @@
   <%= render partial: "layouts/navigation" %>
 </div>
 <div class="org-branding">
-  <% if user_signed_in? && !current_user.org.nil? && !current_user.org.is_other %>
+  <% if user_signed_in? && ((!current_user.org.nil? && !current_user.org.is_other) || current_user.can_super_admin?) %>
```

Comment:
What is the desired behavior here?
I think if the super admin did not have an org, this would still render the page which would break.
Should this be:
` user_signed_in? && !current_user.org.nil? && (current_user.can_super_admin? || !current_user.org.is_other`